README.md

```diff
@@ -2,7 +2,7 @@
 
 Thank you for visiting our repo!  We are a PHP User Group in Raleigh, NC.  We welcome your contributions!  Feel free to submit an issue or a Pull Request.
 
-[Visit our page](https://trianglephp.com)
+[Visit our page](http://trianglephp.com)
```

Comment:
we're `https` now :)

Reply:
I fixed it.  I probably should have used git revert, but I jut removed the commit